### ISSUE TITLE
ComfyUI doesn't start anymore after ComfyUI-Manager update

### ISSUE THREAD

**Issue by MoonMoon82** [Comfy-Org/ComfyUI-Manager#286]
Hi!

After I updated ComfyUI-Manager and some other Nodes, this message appears at the start up:
```
e:\StableDiffusion\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build --listen --dont-upcast-attention --disable-xformers --disable-auto-launch --preview-method auto
** ComfyUI startup time: 2024-01-04 08:27:59.851410
** Platform: Windows
** Python version: 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
** Python executable: e:\StableDiffusion\ComfyUI_windows_portable\python_embeded\python.exe
** Log path: e:\StableDiffusion\ComfyUI_windows_portable\comfyui.log

#######################################################################
[ComfyUI-Manager] Starting dependency installation/(de)activation for the extension

Install: pip packages for 'e:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\stability-ComfyUI-nodes'
WARNING: Ignoring invalid distribution -rotobuf (e:\stablediffusion\comfyui_windows_portable\python_embeded\lib\site-packages)

[notice] A new release of pip is available: 23.2.1 -> 23.3.2
[notice] To update, run: e:\StableDiffusion\ComfyUI_windows_portable\python_embeded\python.exe -m pip install --upgrade pip
Install: pip packages for 'e:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux'
[!] WARNING: Ignoring invalid distribution -rotobuf (e:\stablediffusion\comfyui_windows_portable\python_embeded\lib\site-packages)
Exception in thread Thread-2 (handle_stream):
Exception in thread Traceback (most recent call last):
Thread-1 (handle_stream)  File "threading.py", line 1016, in _bootstrap_inner
:
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "threading.py", line 953, in run
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 88, in handle_stream
      File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 90, in handle_stream
print(prefix, msg, end="", file=sys.stderr)
      File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 191, in write
print(prefix, msg, end="")
      File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 193, in write
self.sync_write(message)
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 203, in sync_write
    self.sync_write(message)
self.last_char = message[-1]  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 203, in sync_write

IndexError    : self.last_char = message[-1]string index out of range

IndexError: string index out of range
[!] WARNING: Ignoring invalid distribution -rotobuf (e:\stablediffusion\comfyui_windows_portable\python_embeded\lib\site-packages)
Exception in thread Thread-4 (handle_stream):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
Exception in thread Thread-3 (handle_stream):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "threading.py", line 953, in run
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 88, in handle_stream
      File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 90, in handle_stream
print(prefix, msg, end="", file=sys.stderr)
      File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 191, in write
print(prefix, msg, end="")
      File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 193, in write
self.sync_write(message)
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 203, in sync_write
    self.sync_write(message)
self.last_char = message[-1]  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 203, in sync_write

IndexError    : self.last_char = message[-1]string index out of range

IndexError: string index out of range
[!] WARNING: Ignoring invalid distribution -rotobuf (e:\stablediffusion\comfyui_windows_portable\python_embeded\lib\site-packages)
Exception in thread Thread-6 (handle_stream)Exception in thread :
Thread-5 (handle_stream)Traceback (most recent call last):
:
  File "threading.py", line 1016, in _bootstrap_inner
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "threading.py", line 953, in run
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 88, in handle_stream
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 90, in handle_stream
        print(prefix, msg, end="", file=sys.stderr)print(prefix, msg, end="")

  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 191, in write
  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 193, in write
    self.sync_write(message)
self.sync_write(message)  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 203, in sync_write

  File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\prestartup_script.py", line 203, in sync_write
    self.last_char = message[-1]
    IndexErrorself.last_char = message[-1]:
string index out of rangeIndexError
: string index out of range
```
Not quite sure if it's an Manager issue, but the output is mentioning ComfyUI-Manager\prestartup_script.py. 
Should I adress this issue to comfyanonymous?

**Comment by ltdrdata** [comment]
Update to latest version. Patched on V1.23.1

**Comment by MoonMoon82** [comment]
Thank you!
